src/pyOpenMS/pxds/OMSSAXMLFile.pxd

```diff
@@ -20,7 +25,15 @@ cdef extern from "<OpenMS/FORMAT/OMSSAXMLFile.h>" namespace "OpenMS":
                   libcpp_vector[ PeptideIdentification ] & id_data,
                   bool load_proteins,
                   bool load_empty_hits) nogil except +
+            # wrap-doc:
+                #   Loads data from a OMSSAXML file
+                #   -----
+                #   :param filename: The file to be loaded
+                #   :param protein_identification: Protein identifications belonging to the whole experiment
+                #   :param id_data: The identifications with m/z and RT
+                #   :param load_proteins: If this flag is set to false, the protein identifications are not loaded
+                #   :param load_empty_hits: Many spectra will not return a hit. Report empty peptide identifications?
```

Comment:
@jpfeuffer  Is that a question? Should not be in the comment I guess.

Reply:
I think it is a question to the developer: Do you want empty identifications?

Reply:
Ok thanks, then let us reformulate that a bit to make it clearer.

Reply:
@oliveralka can you please suggest the changes.